CONTRIBUTING.md

```diff
@@ -228,6 +229,25 @@ as well as helpful tooling that will ease the contribution process. For example,
 [pre-commit](https://pre-commit.com) is used to simplify
 [code review](/docs/project/code_review.md).
 
+#### Using AI-based contribution tools
+
+All submissions to Carbon need to follow our
+[Contributor License Agreement (CLA)](#contributor-license-agreements-clas), in
+which contributors agree that their contribution is an original work of
+authorship. This doesn’t prohibit the use of coding assistance tools, but what’s
+submitted does need to be a contributor’s original creation.
+
+All contributions, regardless of what tools are used, are also still subject to
+normal code review and our
+[guidelines and standards](#contribution-guidelines-and-standards) below.
+
+Additionally, we want to avoid contributions to Carbon to also be viable as
+contributions to LLVM so that we can potentially move things between these
+projects easily where relevant. We selected our license in part for this reason,
```

Comment:
There's a lot of qualifying here ("potentially", "easily", "where relevant"). You might consider either saying something like "easily move things ... where relevant", or "potentially move things ... when relevant", or "can, if needed, easily move things"